generate-manifest.py

```diff
@@ -76,6 +77,34 @@ def edit_manifest(data, arch, branch, runtime_version):
             data['modules'].insert(0, gtk_module)
             break
 
+def sha256(filename):
+    checksum = hashlib.sha256()
+    with open(filename, 'rb') as file:
```

Comment:
For next time: `file` is a built-in in Python, so it's better not to override it